core/src/banking_stage/consumer.rs

```diff
@@ -470,33 +470,40 @@ impl Consumer {
         chunk_offset: usize,
         pre_results: impl Iterator<Item = Result<(), TransactionError>>,
     ) -> ProcessTransactionBatchOutput {
+        // Lock accounts so that other threads cannot encode transactions that
```

Comment:
Follow-up(?): seems these 3 operations are essentially creating a batch (lock, cost model, unlock).
Do you think this makes sense to create a function for?